tools/update_ct_loglist.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
This script should early-exit on error (`set -e`).

Also, it should have a comment explaining what it does and when to run it.

Perhaps we should include `gojsonschema` in boulder-tools? It doesn't get run as part of our tests but it would be one way to make sure we're all getting a specific version.

Reply:
Done! I've added it to boulder-tools in this PR, but haven't yet built and uploaded a new version of the image.